app/src/main/java/com/duckduckgo/app/tabs/ui/TabSwitcherActivity.kt

```diff
@@ -341,6 +351,11 @@ class TabSwitcherActivity : DuckDuckGoActivity(), TabSwitcherListener, Coroutine
     override fun onPrepareOptionsMenu(menu: Menu?): Boolean {
         val closeAllTabsMenuItem = menu?.findItem(R.id.closeAllTabs)
         closeAllTabsMenuItem?.isVisible = viewModel.tabs.value?.isNotEmpty() == true
+        val duckChatMenuItem = menu?.findItem(R.id.duckChat)
+        runBlocking {
+            duckChatMenuItem?.isVisible = duckChat.showInBrowserMenu()
+        }
```

Comment:
This will probably cause ANRs, would it be possible to cache this value instead of using `runBlocking`?